### PR TITLE
Add poetry install logs to artifact

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .hooks/bootstrap [hunk 1]
```diff
@@ -81,6 +81,8 @@ else
         else
           exit_on_error 1 "Poetry version $(poetry --version) doesn't match the required version: ${POETRY_VERSION}"
         fi
+        cp /builds/xdr/cortex-content/content/poetry-installer-error-*.log ${ARTIFACTS_FOLDER}
+        exit_on_error $error_code "Failed to install Poetry version:${POETRY_VERSION}"
```
Comment (kobymeir) [inline .hooks/bootstrap:85]:
```suggestion
         if [ -n "${ARTIFACTS_FOLDER}" ]; then
        cp ${CI_PROJECT_DIR}/poetry-installer-error-*.log "${ARTIFACTS_FOLDER}"
        fi
        exit_on_error $error_code "Failed to install Poetry version:${POETRY_VERSION}"
```

File: .hooks/bootstrap [hunk 2]
```diff
@@ -72,7 +72,7 @@ else
     if [[ "${should_install_poetry}" == "yes" ]]; then
         echo "Installing Poetry version:${POETRY_VERSION}"
         curl -sSL https://install.python-poetry.org | python3 - --version "${POETRY_VERSION}"
-        exit_on_error $? "Failed to install Poetry version:${POETRY_VERSION}"
+        error_code = $?
```
Comment (kobymeir) [inline .hooks/bootstrap:75]:
```suggestion
        error_code=$?
```
no spaces in bash assignment

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: .hooks/bootstrap [hunk 3]
```diff
@@ -81,6 +81,10 @@ else
         else
           exit_on_error 1 "Poetry version $(poetry --version) doesn't match the required version: ${POETRY_VERSION}"
         fi
+        if [ -n "${ARTIFACTS_FOLDER}" ]; then
+            cp "${CI_PROJECT_DIR}/poetry-installer-error-*.log" "${ARTIFACTS_FOLDER}"
```
Comment (kobymeir) [inline .hooks/bootstrap:85]:
```suggestion
            cp ${CI_PROJECT_DIR}/poetry-installer-error-*.log "${ARTIFACTS_FOLDER}"
```

**Review by kobymeir — CHANGES_REQUESTED** [review]

**Review by kobymeir — APPROVED** [review]

**Comment by MosheEichler** [comment]
[Triggered](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/790654) SDK nightly

**Pull request merged**